i6_models/config.py

```diff
@@ -0,0 +1,51 @@
+from dataclasses import MISSING, dataclass
+from typing import Any, List, Optional
+
+
+@dataclass
+class ModelConfiguration:
+    """base dataclass that supported fetching attributes"""
+
+    _name: Optional[str] = None
+
+    @staticmethod
+    def name() -> Optional[str]:
+        return None
```

Comment:
If others agree this can def. be changed. For now this is just similar to the Fairseq reference implementation. But I assume that in this case yes each config should overwrite this if use is desired.

Reply:
For most of the functions here (and the ones I dropped) I found that they are barely used, but still I thought they provide nice optional functionality.

Reply:
But still, I don't understand the purpose of this function. It just returns `None`. So is this supposed to be overwritten? I don't understand the purpose. It this just for debugging purpose, or does this have any actual semantic meaning?

Reply:
So this is what I understand. But then I was not able to find a Config which overwrites this, so maybe we can remove it for now.

Reply:
Yes, I would remove code which we don't need for now. Once we need it, we can add it.